x-pack/plugins/observability_solution/investigate_app/public/pages/details/components/investigation_timeline/investigation_timeline.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useRef } from 'react';
+import moment from 'moment';
+
+import { Chart, Axis, AreaSeries, Position, ScaleType, Settings } from '@elastic/charts';
+import { useActiveCursor } from '@kbn/charts-plugin/public';
+import { EuiSkeletonText } from '@elastic/eui';
+import { getBrushData } from '@kbn/observability-utils/chart/utils';
+import { AnnotationEvent } from './annotation_event';
+import { TIME_LINE_THEME } from './timeline_theme';
+import { useFetchEvents } from '../../../../hooks/use_fetch_events';
+import { useInvestigation } from '../../contexts/investigation_context';
+import { useKibana } from '../../../../hooks/use_kibana';
+import { AlertEvent } from './alert_event';
+
+export const InvestigateTimeLine = () => {
```

Comment:
how about `EventsTimeLine` since these are the events that are generally detected in infrastructure and we filter them down for the investigation? `InvestigationTimeline` might have different meaning depending on context - e.g. timeline of visualizations/notes added by user in the investigation